### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by Wikitelmex**
# Contact form
- add contact form
- add contact form styel
- add social icon bars to contact form
- styling social bar and background

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,238 +1,264 @@
...
+                    </li>
+                </ul>
+            </section>
+        </div>
+        <footer>
+            <div class="light">
+                <h1 id="h1footer">
+                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                </h1>
+                <form name="contact-form" class="portfolio-form" action="https://formspree.io/f/mnqlovey">
```
Comment (karvel-code) [inline index.html]: - [x] Please ensure you add the attribute `method = "POST" `. It is important as it tells the web browser how to send form data to a server.

File: index.html [hunk 2]
```diff
@@ -1,238 +1,264 @@
...
+                    <div class="wrap">
+                        <div class="pull-tab"></div>
+                        <textarea class="textarea" name="contact-textarea" cols="30" rows="10"></textarea>
                     </div>
-                    <div class="card-footer">
-                        <a href="#">See Project</a>
-                    </div>
+                    
+                    <button class="dark-button" type="submit">Get It Touch</button>
+                </form>
```
Comment (karvel-code) [inline index.html]: - [x] Your form does not submit data successfully. The issue could be that the attribute method = "POST" is missing. I'd advise you first to add it and test if your form works well when valid data is submitted.

File: index.html [hunk 3]
```diff
@@ -1,238 +1,264 @@
...
+                <h1 id="h1footer">
+                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                </h1>
+                <form name="contact-form" class="portfolio-form" action="https://formspree.io/f/mnqlovey">
+                    <input class="textbox" type="text" placeholder="Full name" required>
+                    <input class="textbox" type="email" placeholder="Email address" required>
+                    
+                    <div class="wrap">
+                        <div class="pull-tab"></div>
+                        <textarea class="textarea" name="contact-textarea" cols="30" rows="10"></textarea>
```
Comment (karvel-code) [inline index.html]: - [x] Please add a placeholder attribute for the `textarea` tag. It's important as it informs the user what is expected of him/her.
- [x] Please add the `required` attribute for the `textarea` tag. It enhances the validations for the input.

File: index.html [hunk 4]
```diff
@@ -1,238 +1,264 @@
...
+                </ul>
+            </section>
+        </div>
+        <footer>
+            <div class="light">
+                <h1 id="h1footer">
+                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                </h1>
+                <form name="contact-form" class="portfolio-form" action="https://formspree.io/f/mnqlovey">
+                    <input class="textbox" type="text" placeholder="Full name" required>
```
Comment (karvel-code) [inline index.html]: - [x] Please add a `maxlength="30"` attribute for the name input tag. It is important in order to limit the number of characters(in this case to 30) a user can input for this input tag.

File: index.html (hunk 3 above)
Comment (karvel-code) [reply index.html]: - [x] Please add a `maxlength="500"` attribute for the textarea tag

**Review by karvel-code — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All your linter checks pass successfully.

## Required Changes ♻️
_Check the comments under the review._

- [x] Please eliminate the dark part appearing on the right edge of the form section as it is not provided in the Figma design.
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 2 above)
Comment (Wikitelmex) [reply index.html]: - modifications added and started to work well

**Review by AshakaE — APPROVED** [review]
### Hi @Wikitelmex  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**